src/components/details/template.test.js

```diff
@@ -0,0 +1,80 @@
+/* globals describe, it, expect */
+
+const { render, getExamples } = require('../../../lib/jest-helpers')
+
+const examples = getExamples('details')
+
+describe('Details', () => {
+  it('renders a details element', () => {
+    const { $ } = render('details', examples.default)
+
+    const $component = $('.govuk-c-details')
+    expect($component.get(0).tagName).toEqual('details')
+  })
+
+  it('includes a nested summary', () => {
+    const { $ } = render('details', examples.default)
+
+    // Look for the summary element within the details element
+    const $summary = $('.govuk-c-details__summary', '.govuk-c-details')
```

Comment:
Didn't even know you could do this, I've been doing `$('...').find('...')` all this time.

Reply:
Yeah, I think it's called 'context' in jQuery-land. I've seen in the past people generally aren't that aware of it – hence the comment. Do you think it's clear enough what's it doing?

Reply:
In this specific case I'd just rely on good ol' CSS selectors `$('.govuk-c-details > .govuk-c-details__summary')` which would hopefully be more widely understood.

Reply:
That's a good call, though not sure we'd want to use `>` as that would be testing that the summary is a direct descendant - which I don't think we care about?

Reply:
I stealth removed the direct descendant, but wasn't quick enough 😢

Reply:
😁